ibm/service/kms/resource_ibm_kms_key.go

```diff
@@ -84,6 +84,12 @@ func ResourceIBMKmskey() *schema.Resource {
 				ValidateFunc: validate.ValidateAllowedStringValues([]string{"public", "private"}),
 				Description:  "public or private",
 			},
+			"description": {
+				Type:        schema.TypeString,
+				Optional:    true,
+				ForceNew:    true,
```

Comment:
Change in description is forceNew?
We can't update description of the key?

Reply:
Yes, this is actually correct. Key Protect does not have a PATCH endpoint for keys.

Reply:
Also if you look at the KMSKeyUpdate function it doesnt do anything really